### ISSUE TITLE
Heading tag should not be used as title of the blocks/admonitions

### ISSUE THREAD

**Issue by amimas** [favoloso/remarkable-admonitions#43]
The plugin currently generates an html code where the title of the special notes/admonitions are given an html heading `h5`.

https://github.com/favoloso/remarkable-admonitions/blob/042d6e109eee77fb308a79935cc4f696880b0156/src/renderer.ts#L19-L25

This is an issue from both semantics and accessibility aspect. If I have the following structure in my markdown:

```md

# My page/document title

## Section heading

Some introduction about this heading and here's a special note:

:::caution Something to keep in mind
some text here
:::

...
```

The above will produce the following hierarchy when the html page is generated:

```
h1 My page/document title

h2 Section heading

Some introduction about this heading and here's a special note:

h5 Something to keep in mind
some text here
...
```

Semantically that's inaccurate because after h2, the next sub-heading should be h3. For the same reason, it also fails the accessibility tests in Google's Lighthouse audit.

I think this plugin should not use `h5` as the title of the notes. It can probably use a `span` (or something more appropriate) with a classname and then adjust the font size accordingly.

**Comment by leonardfactory** [comment]
Seems right to me. Maybe I'll add an option to revert to previous behaviour if someone has an issue with this (even if I don't see how).

**Comment by amimas** [comment]
Sounds good. I didn't know this was changed in new version. I'd probably go for reverting this change instead of adding option to maintain something that is not correct.
